lib/audit-ci-reporter.js

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+const childProcess = require('child_process');
+const reporter = require('npm-audit-report');
+
+/**
+ * @param {object} npmAudit The NPM audit
+ * @param {{report: boolean, whitelist: string[],
```

Comment:
It is unfortunate that this syntax gets highlighted on both GitHub and VS Code. If I bring the line below onto the same line, it is a `max-len` issue that I can't suppress on a per-line basis. Consider being less specific in the object type to fit onto one line.

Reply:
This has been addressed by disabling ESLint `max-len` for `audit-ci-reporter.js`.